### PR TITLE
fix: define `research` and `promotion` contribution types

### LINKED ISSUE #399: `research` contribution key is not recognized
**Describe the bug**
Research contribution key is not recognized.

**Expected behavior**
research is a valid key: https://allcontributors.org/docs/en/emoji-key

**Screenshots**
image

### PR DESCRIPTION
**Pull request by 201B153**
- We are adding research and promotion keywords to parse-comment.js 
- for Issue #399

**What**: Adding new keywords to parse-comment.js

**Checklist**:

- [ ] Documentation
- [ ] Ready to be merged 
- [ ] Added myself to contributors table. 
[Bot Usage](https://allcontributors.org/docs/en/bot/installation#4-update-your-contributing-documentation)

### CODE REVIEW

**Review by tenshiAMD — CHANGES_REQUESTED** [review]
@201B153 Looks like something is missing. where do we add the emoji?

**Comment by 201B153** [comment]
> @201B153 Looks like something is missing. where do we add the emoji?

Do we have to add emojis too??
where I don't read it in the issue can u guide me here?

**Review by tenshiAMD — APPROVED** [review]
> Do we have to add emojis too?? where I don't read it in the issue can u guide me here?
> 
> image

@201B153 Thanks for this one. LGTM now. I just noticed now that the emoji keys are in the CLI codebase. 😕 I think will do some refactoring later on regarding this one.

**Comment by 201B153** [comment]
> @201B153 Thanks for this one. LGTM now. I just noticed now that the emoji keys are in the CLI codebase. 😕 I think will do some refactoring later on regarding this one.

Ok, Thanks

File: lib/parse-comment.js [hunk 1]
```diff
@@ -35,6 +35,8 @@ const validContributionTypes = [
   "tutorial",
   "usertesting",
   "video",
+  "research",
+  "promotion",
```
Comment (tenshiAMD) [inline lib/parse-comment.js:39]: TODO: Add an emoji key for `promotion` in https://github.com/all-contributors/cli

**Review by tenshiAMD — CHANGES_REQUESTED** [review]
Need to complete all todos.

File: lib/parse-comment.js (hunk 1 above)
Comment (201B153) [reply lib/parse-comment.js:39]: U mean adding emoji in docs here 
> https://github.com/all-contributors/all-contributors/blob/master/docs/emoji-key.md

**Comment by tenshiAMD** [comment]
@all-contributors add @201B153 for code

**Pull request merged**